### DISCUSSION TITLE
You are currently running a version of TypeScript which is not officially supported

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#22861] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

Invoking `yarn test` in dev container.

```
root@f5fff38bf81a:/workspaces/renovate# yarn test
yarn run v1.22.19
$ run-s generate:* 
$ node tools/generate-imports.mjs
generating imports
> data/azure-pipelines-tasks.json
> data/debian-distro-info.json
> data/kubernetes-api.json5
> data/node-js-schedule.json
> data/ubuntu-distro-info.json
> node_modules/emojibase-data/en/shortcodes/github.json
generating hashes
$ run-s lint test-schema jest
$ run-s ls-lint type-check eslint prettier markdown-lint git-check doc-fence-check
$ ls-lint
$ run-s generate:* "tsc --noEmit {@}" --
$ node tools/generate-imports.mjs
generating imports
> data/azure-pipelines-tasks.json
> data/debian-distro-info.json
> data/kubernetes-api.json5
> data/node-js-schedule.json
> data/ubuntu-distro-info.json
> node_modules/emojibase-data/en/shortcodes/github.json
generating hashes
$ tsc --noEmit
$ eslint . --cache --cache-location .cache/eslint --report-unused-disable-directives
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <5.1.0

YOUR TYPESCRIPT VERSION: 5.1.3

Please only submit bug reports when using the officially supported version.

=============
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by viceice** [comment, marked as answer]
ignore it, it still works. typescript eslint is not yet officially compatible with typescript 5.1

**Reply by lmilbaum** [reply]
Thanks @viceice for your response. Personally, I don't like warnings. I treat them as errors.

**Reply by viceice** [reply]
there isn't much we can do. it's just an eslint warning and we don't like to manage the typescript version manually because of some peer dependencies. 🤷‍♂️
